### ISSUE TITLE
polymer_ui_collapsable throws on content click

### ISSUE THREAD

**Issue by tusj** [bwu-dart-graveyard/polymer_ui_elements#165]
I wanted to have a set of buttons which I could show / hide and thought that this element would fit. However, it is not possible to disable show / hide behaviour of the content, only the header. So what happens is that a button click also toggles the view. This is actually what I want, but it throws when I click on my button:

```
NoSuchMethodError : method not found: 'notap='
Receiver: Instance of 'MouseEvent'
Arguments: [true]
Exception: Class 'MouseEvent' has no instance setter 'notap='.
```

Because I am quite new to polymer I am not sure if this is caused due to some of my implementation issues.

**Comment by zoechi** [comment]
Can you provide some code? This could be a problem with polymer_ui_elements. At several places there is used the 'tap' event in the original JavaScript elements but in Dart the 'tap' event doesn't work so we use 'click' instead. It may be that we have overlooked something.

**Comment by tusj** [comment]
So in `"content"` there are a set of buttons for which I'd like to be able to click.

```
<polymer-ui-collapsible>
  <div class="polymer-ui-collapsible-header">Login</div>
  <div class="content">
  <button ng-repeat="login in bd.logins" ng-click="bd.login(login)">{{login.serviceName}}</button>
  </div>
  </polymer-ui-collapsible>
</polymer-ui-collapsible>
```

**Comment by zoechi** [comment]
I think I found the bug. I fix it and publish a new release. What Dart/Polymer version are you using?
The current release requires Dart 1.3 and Polymer 0.10.0-pre.8.

**Comment by zoechi** [comment]
I published a new release 0.2.4-pre.4. 
You need to set a very specific version constraint to make pub upgrade take pre-releases. like ">=0.2.0-pre.4"

**Comment by tusj** [comment]
Thanks! I think I tried this on 1.3 as I believe polymer complained that the 1.2 sdk didn't meet the requirements. I'd prefer to wait for testing the fix as I normally don't use the development release.
